recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set version = "16.0.1" %}
 # check https://clang.llvm.org/docs/OpenMPSupport.html
 # occasionally to see last fully supported openmp ver.
-{% set openmp_ver = "4.5" %}
+{% set openmp_ver = "5.1" %}
```

Comment:
Did you read the comment immediately above the version? LLVM is far from full support for 5.0, much less 5.1

Reply:
D'oh! This is the most embarrassing PR I've ever made in my life.

Reply:
Then again, what do you expect from a guinea pig with glasses :)